spec/models/user_spec.rb

```diff
@@ -61,5 +64,23 @@
         end
       end
     end
+
+    context 'birth validations' do
+      context 'when the age of the user is under 18' do
+        let(:birth) { '2010-03-01' }
```

Comment:
this test is only valid up to a specific date, which is `01/03/2028`. on the second of march of 2028, this test will fail. The value should be generated based off the 
current date - (18 years + some random value of days/months/years)